subsys/bluetooth/host/uuid.c

```diff
@@ -98,15 +98,14 @@ bool bt_uuid_create(struct bt_uuid *uuid, const u8_t *data, u8_t data_len)
 	return true;
 }
 
-#if defined(CONFIG_BT_DEBUG)
 void bt_uuid_to_str(const struct bt_uuid *uuid, char *str, size_t len)
 {
 	u32_t tmp1, tmp5;
 	u16_t tmp0, tmp2, tmp3, tmp4;
 
 	switch (uuid->type) {
 	case BT_UUID_TYPE_16:
-		snprintk(str, len, "%04x", BT_UUID_16(uuid)->val);
+		snprintk(str, len, "%02x", BT_UUID_16(uuid)->val);
```

Comment:
This change looks both unrelated and wrong (the precision indicates outputted characters, not size-in-bytes of the integer)

Reply:
True, but then the TYPE_32 should have 0x08 then.